qiita_db/artifact.py

```diff
@@ -1115,23 +1119,98 @@ def descendants_with_jobs(self):
             sql = """SELECT *
```

Comment:
Can you list the columns that you are retrieving? I'm mainly concern about the order, which is assumed in the for loop

Reply:
Done